exporter/prometheusremotewriteexporter/exporter.go

```diff
@@ -130,12 +131,19 @@ func (prwe *prwExporter) PushMetrics(ctx context.Context, md pmetric.Metrics) er
 	case <-prwe.closeChan:
 		return errors.New("shutdown has been called")
 	default:
+
 		tsMap, err := prometheusremotewrite.FromMetrics(md, prwe.exporterSettings)
 		if err != nil {
 			err = consumererror.NewPermanent(err)
 		}
+
+		if prwe.exporterSettings.SendMetadata {
+			m := prometheusremotewrite.OtelMetricsToMetadata(md, prwe.exporterSettings.AddMetricSuffixes)
+			// Call export even if a conversion error, since there may be points that were successfully converted.
+			return multierr.Combine(err, prwe.handleExport(ctx, tsMap, m))
+		}
```

Comment:
```suggestion
		var m []*prompb.MetricMetadata
		if prwe.exporterSettings.SendMetadata {
			m = prometheusremotewrite.OtelMetricsToMetadata(md, prwe.exporterSettings.AddMetricSuffixes)
		}
```

let's not duplicate the export/error handling code if we can to avoid potential errors resulting from changes needing to be made in multiple places.